index.src.html

```diff
@@ -891,9 +920,9 @@ <h4 id="algorithm-require-mediation" algorithm>Require User Mediation</h4>
           var c = new <a idl lt="PasswordCredential(form)">PasswordCredential</a>(e.target);
 
           // Fetch the form's action URL, passing that new credential object in
-          // as the fetch's credentials. If the response indicates success, ask
+          // as a FormData object. If the response indicates success, ask
           // the user agent to ask the user to store the credential for future use:
-          var init = { method: "POST", credentials: c };
+          var init = { method: "POST", body: new FormData(e.target) };
```

Comment:
We should note somewhere the encoding restrictions on this mechanism. Maybe add an issue to https://w3c.github.io/webappsec-credential-management/#implementation-authors?